electron/src/components/privilege-checker/privilege-checker.component.ts

```diff
@@ -81,4 +90,32 @@ export class PrivilegeCheckerComponent {
         return subject.asObservable();
     }
 
+    private setUpPermissionsOnWin(): Observable<void> {
+        const subject = new ReplaySubject<void>();
+        /**
+         * source code: https://github.com/pbatard/libwdi
+         */
+        const scriptPath = path.resolve(this.rootDir, `rules/zadic-${process.arch}.exe`);
+        const options = {
+            name: 'Setting UHK access rules'
+        };
+        /**
+         * The parameters:
+             - vid: vendor ID
+             - pid: product ID
+             - iface: interface ID
+             - usealldevices: if the device has installed USB driver than overwrite it
+             - noprompt: return at the end of the installation and not waiting for any user command
+         */
+        const command = `"${scriptPath}" --vid 0x1D50 --pid 0x6122 --iface 0 --usealldevices --noprompt`;
```

Comment:
The VID and PID values are also used elsewhere in the codebase. Please extract them and reference them, so they won't be duplicated.